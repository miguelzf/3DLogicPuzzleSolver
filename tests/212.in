- - - - -
- - - - X
- - R - -
- - - X -
- - - - -

- - - - X
- - - - -
- - - - -
- - - Y -
- - - - -

L - - - -
- X X X X
- - - - -
- R X - -
- - - L Y
