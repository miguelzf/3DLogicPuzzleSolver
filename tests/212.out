
Solution:
l  l  l  l  l  
l  y  y  y  X  
l  y  R  y  y  
l  y  r  X  y  
l  y  r  y  y  
=============
l  y  r  y  X  ||  L  l  l  l  l  
l  y  r  y  y  ||  y  X  X  X  X  
l  y  r  r  r  ||  y  y  y  y  y  
l  y  y  Y  r  ||  r  R  X  -  y  
l  l  l  l  l  ||  l  l  l  L  Y  
